dm/syncer/err-operator/operator.go

```diff
@@ -32,17 +34,20 @@ import (
 // Operator contains an operation for specified binlog pos
 // used by `handle-error`.
 type Operator struct {
-	uuid   string // add a UUID, make it more friendly to be traced in log
-	op     pb.ErrorOp
-	events []*replication.BinlogEvent // startLocation -> events
+	uuid          string // add a UUID, make it more friendly to be traced in log
+	op            pb.ErrorOp
+	isAllInjected bool                       // is all injected, used by inject
+	events        []*replication.BinlogEvent // startLocation -> events
+	originReq     pb.HandleWorkerErrorRequest
 }
 
 // newOperator creates a new operator with a random UUID.
-func newOperator(op pb.ErrorOp, events []*replication.BinlogEvent) *Operator {
+func newOperator(op pb.ErrorOp, events []*replication.BinlogEvent, originReq pb.HandleWorkerErrorRequest) *Operator {
```

Comment:
```suggestion
func newOperator(op pb.ErrorOp, events []*replication.BinlogEvent, originReq *pb.HandleWorkerErrorRequest) *Operator {
```

maybe

Reply:
I have used pointer before, but it occurs error, maybe the request was modified. i will test again.